### ISSUE TITLE
Visual regression testing

### ISSUE THREAD

**Issue by Windvis** [appuniversum/ember-appuniversum#263]
A lot of components have arguments to change the styles of the component. This can't really be integration tested without checking css classes and even then the implementation can change by accident.

For those cases it might be a good idea to setup some kind of visual regression testing setup. Visual changes need to be approved so we never accidentally break styles.

[BackstopJS](https://github.com/garris/BackstopJS) is an open-source solution for this but there migth be others.

**Comment by Dietr** [comment]
Good idea. I tried [chromatic](https://www.chromatic.com/) before but having some issues on making it work with our ember setup and also not open source.

**Comment by Dietr** [comment]
Figured out how to use chromatic and did a test with it: https://www.chromatic.com/library?appId=61e5835e5fbce3003a653ee6

Is this an option or do we look into one of the other tools?

**Comment by Windvis** [comment]
@Dietr Cool! I think it's a good start and we can reevaluate in the future? How does the integration work exactly?

**Comment by Dietr** [comment]
@Windvis I created a PR (https://github.com/appuniversum/ember-appuniversum/pull/267) with the workflow. Feel free make changes.
Current logic is when a PR is created it checks it for visual changes against the dev branch. If there a no changes the check completes. If there are changes you have to approve them in chromatic.
